VideoWeb/VideoWeb/ClientApp/src/app/shared/self-test/self-test.component.ts

```diff
@@ -258,7 +258,7 @@ export class SelfTestComponent implements OnInit, OnDestroy {
             participant: this.selfTestParticipantId,
             pexipDisconnectReason: reason
         });
-        if (reason.reason === 'Conference terminated by another participant') {
+        if (reason.reason === 'Conference terminated by another participant' || reason.reason === 'Test call finished') {
```

Comment:
For readability, put the reasons into an array labelled `expectedDisconnectReasons` or whatever you think sounds right for this scenario